pkg/controller/workload/job/job_webhook_test.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package job
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	batchv1 "k8s.io/api/batch/v1"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+
+	testingutil "sigs.k8s.io/kueue/pkg/util/testing"
+)
+
+func TestValidateUpdate(t *testing.T) {
```

Comment:
Also add a test for create

Reply:
We didn't have validation on creation now. All jobs will be suspend in defaulting if met the requirements.

Reply:
But we can still test that suspend is set to true